tools/replaceVersion.sh

```diff
@@ -8,9 +8,7 @@ ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/..
 declare -a folders=(
     "$ROOT/examples"
     "$ROOT/gradle-plugins"
-    "$ROOT/components"
```

Comment:
We hardcode Kotlin in components and html now, to support both older and newer Kotlin's